hera_notebook_templates/cli.py

```diff
@@ -70,10 +69,14 @@ def runfunc(ctx, basename, **kwargs):
         output_path = Path(ctx.obj['output_dir']) / f"{basename}.ipynb"
 
         if ctx.obj['toml'] is not None:
-            toml = toml.load(ctx.obj['toml'])
+            cfg = toml.load(ctx.obj['toml'])
```

Comment:
This code is basically uncommented. It'd be nice to include some comments for those of less familiar with `click` and `papermill`

Reply:
I've included some comments, thanks for the suggestion.

Reply:
Not seeing that... did you push?